docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services: 
+  fluid-queue:
+    container_name: fluidQueue
+    build: 
+      context: ./.
+    image: fluidQueue:latest
+    volumes:
+      - ./data:/app/data/
```

Comment:
We also need to mount the settings folder here to persist aliases, or your aliases will disappear when the ephemeral volume is removed.

Reply:
done.